ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -152,6 +154,28 @@
         (get-unique-native-id context subscription))
       native-id)))
 
+(defn add-id-and-email-to-metadata-if-missing
```

Comment:
For private functions that are only invoked within the namespace, use `defn-`.

Reply:
👍